Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -437,4 +444,17 @@ class NewsTableViewController: UITableViewController,
     func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
         return DimPresentAnimationController()
     }
+    
+    // MARK: SFSafariViewControllerDelegate
+    func safariViewController(_ controller: SFSafariViewController,
+                              activityItemsFor URL: URL,
+                              title: String?) -> [UIActivity] {
+        
+        guard let n = self.selectedNews else {
+            return []
+        }
+        
+        let activity = ShareCanillitapActivity(withNews: n)
+        return [activity]
+    }
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/e02648ce2406a2f85d493d64af3d34cc6c9bd946/Headlines/src/Controllers/NewsTableViewController.swift#L460"></span>File should contain 400 lines or less: currently contains 460</td>
    </tr>
  </tbody>
</table>